thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -24,6 +26,18 @@ const isValid = (key, value) => {
       return value && value.length;
     case 'compareMode':
       return ['WoW', 'Wo2W', 'Wo3W', 'Wo4W'];
+    case 'metricId':
```

Comment:
fyi, you can aggregate cases like so: 
```
  case 'metricId':
  case 'anomalyId':
    return !value || (Number.isInteger(value) && parseInt(value) >= 0);
```

Reply:
yes. this is for consistency with other uses of switch() in our project